example_project_app/plugins.py

```diff
@@ -19,6 +19,21 @@
 )
 
 
+def get_example_setting_default(project=None, user=None):
+    """Example callable function for different scopes
```

Comment:
Just a style note here: Docstrings should have line break after the initial `"""`, also an empty line between the description and params/return value. Otherwise they will not render correctly when building API docs with Sphinx.